libopenage/coord/phys.h

```diff
@@ -74,11 +77,11 @@ namespace std {
 template<>
 struct hash<openage::coord::phys3> {
 	size_t operator ()(const openage::coord::phys3 &pos) const {
-		return (
-			openage::util::rol<size_t, 2>(hash<openage::coord::phys_t>{}(pos.ne)) ^
-			openage::util::rol<size_t, 1>(hash<openage::coord::phys_t>{}(pos.se)) ^
-			                              hash<openage::coord::phys_t>{}(pos.up)
-		);
+		size_t hash = std::hash<std::type_index>()(std::type_index(typeid(openage::coord::phys3)));
```

Comment:
as we invoke this function multiple times over the codebase and it seems quite verbose, we could add a helper function to `../util/hash.h` that looks like:
``` cpp
template <typename T>
type_hash() {
    return std::hash<std::type_index>()(std::type_index(typeid(T)));
}
```

Reply:
Done, juchu!